server/src/main/java/io/spine/server/ServerEnvironment.java

```diff
@@ -224,122 +223,6 @@ public void configureDeployment(Supplier<DeploymentType> supplier) {
         deploymentDetector = supplier;
     }
 
-    /**
-     * Assigns the specified {@code TransportFactory} for the specified application environment.
-     *
-     * @return this instance of {@code ServerEnvironment}
-     * @deprecated please use {@link #when(Class) when(Class<? extends EnvironmentType>)}
-     *         followed by {@link TypeConfigurator#use(StorageFactory) use(StorageFactory)}
-     */
-    @Deprecated
```

Comment:
Generally, I'm OK with the removal of these deprecated methods, but we have deprecated them in 1.6.x **and** removed them in 1.6.x. We must at least not to forget to include this backward-incompatible change to the release notes.

Reply:
To be addressed via #1337.